classes/controller/AdminController.php

```diff
@@ -4882,4 +4894,30 @@ private function prepareNumberSpecifications(Context $context)
             $numberSpecification->toArray()
         );
     }
+
+    /**
+     * Using this method will display the given message as an "alert" message at the top of the current
+     * administration page.
+     *
+     * @see admin-dev/themes/default/template/content.tpl
+     *
+     * @param string $message The message to be displayed
+     * @param string $namespace Type of alert, use FRONT_ALERT_XXX constants
+     */
+    protected function addTopAlertMessage($message, $namespace = false)
```

Comment:
```suggestion
    protected function addTopAlertMessage($message, $namespace = self::FRONT_ALERT_INFO)
```
and remove the `if (!$namespace) {` condition :)

Reply:
Good tip ! I was so persuaded I couldn't use a constant as a default parameter...

Reply:
AS Pierre suggested you can use the constant directly as a default value

Reply:
done !